docs/camino-messenger/introduction.md

```diff
@@ -74,7 +68,14 @@ Just like any API end-point from the web2 era, you can check the availability of
 
 #### Partners, Supplier Fee and Network Fee
 
-Partner configuration can be managed via the Camino Partner Configurator, which will form part of the Camino Application Suite. Two messages have been made available to enable automatic detection of changed partners, their settings and fees:
+Partner configuration can be managed via the Camino Partner Configurator, which will form part of the Camino Application Suite. The Partner Config stores into a smart contract CM Account:
+
+- Supported Services (full package name including the version, rack-rate flag, fee, capabilities) -- for suppliers to specify which services they support, whether they have a rack-rate or public rate that any Camino Partner can book without prior agreement, what fee they charge to respond to (search) messages and to specify particularities of their integration in capabilities.
+- Wanted Services (full package name including the version) -- for distributors to specify which services they support and search for when matching with others
+- Payment info (supported currencies: Off-Chain, CAM, stable coins)
```

Comment:
Instead of stable coins, we should say ERC20 tokens (ex: stable coins). We didn't restrict it to be only stable coins.

Reply:
ERC20 tokens added, so that we still mention stable coins, which is good for the travel industry